src/Api/SecretsManager/Controllers/SecretsManagerPortingController.cs

```diff
@@ -31,7 +31,7 @@ public SecretsManagerPortingController(ISecretRepository secretRepository, IProj
     }
 
     [HttpGet("sm/{organizationId}/export")]
-    public async Task<SMExportResponseModel> Export([FromRoute] Guid organizationId, [FromRoute] string format = "json")
```

Comment:
I think it would make more sense to me to remove it until we tackle the format import/export, yes. As you mention we might end up doing it client side and leaving this parameter unused. That said I don't have a strong opinion either way, I just removed it as it broke the Swagger generator (Should probably be `[FromQuery]` instead of `[FromForm]` as it is now)

Reply:
Sounds good. Let's remove it in this PR (how you have it), and I have created the PR for the clients side: https://github.com/bitwarden/clients/pull/5470

Reply:
FYI @dani-garcia, the aforementioned [PR](https://github.com/bitwarden/clients/pull/5470) has been merged 👍